stable/bundle_dep.pony

```diff
@@ -50,3 +52,57 @@ class BundleDepGitHub
     if git_tag isnt None then
       Shell("cd " + root_path() + " && git checkout " + (git_tag as String))
     end
+
+class BundleDepLocalGit
+  let bundle: Bundle
+  let info: JsonObject box
+  let package_name: String
+  let local_path: String
+  let git_tag: (String | None)
+  new create(b: Bundle, i: JsonObject box)? =>
+    bundle       = b
+    info         = i
+    package_name = try info.data("package-name") as String
```

Comment:
The package_name is only used for naming the subdirectory in .deps.  There's probably a better way of doing this.